### DISCUSSION TITLE
Imputation in Treesequence.write_vcf() ?

### DISCUSSION THREAD

**Discussion by chlangley** [tskit-dev/tskit#2078] [Q&A] [answered]
I see no mention of missing data in the documentation of write_vcf().  
Perhaps there is a way control whether missing data genotypes are imputed or left in the vcf output?

**Comment by benjeffery** [comment, marked as answer]
Hi @chlangley!
The `write_vcf` code doesn't yet support missing data, and will error if asked to output it. The work to support missing data isn't currently scheduled, but we're always open to contributions if you'd like to have a go!

(I'm moving this to the tskit repo as that's where this code lives)
